src/homepage.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable no-restricted-syntax */
+
+import Meal from './meal.js';
+
+export function updateMealsNumber(mealsList) {
```

Comment:
- [ ] It looks like here we could use ES6 best practices (in this case arrow function). 🐱